src/main/java/at/hannibal2/skyhanni/utils/TimeOffset.kt

```diff
@@ -0,0 +1,56 @@
+package at.hannibal2.skyhanni.utils
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.DebugDataCollectEvent
+import at.hannibal2.skyhanni.events.ProfileJoinEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.TimeUtils.format
+import kotlinx.coroutines.launch
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import org.apache.commons.net.ntp.NTPUDPClient
+import java.net.InetAddress
+import kotlin.math.abs
+import kotlin.math.absoluteValue
+import kotlin.time.DurationUnit
+import kotlin.time.toDuration
+
+@SkyHanniModule
+object TimeOffset {
```

Comment:
please rename to ComputerTimeOffset or LocalTime or LocalComputerTime or something to be a bit more descriptive